narwhal/types/src/primary.rs

```diff
@@ -650,11 +794,12 @@ impl Certificate {
             DagError::CertificateRequiresQuorum
         );
 
+        // TODO(nkls): work out how to do this.
         // Verify the signatures
-        let certificate_digest: Digest<{ crypto::DIGEST_LENGTH }> = Digest::from(self.digest());
-        self.aggregated_signature
-            .verify(&pks[..], certificate_digest.as_ref())
-            .map_err(|_| DagError::InvalidSignature)?;
+        //  let certificate_digest: Digest<{ crypto::DIGEST_LENGTH }> = Digest::from(self.digest());
+        //  self.aggregated_signature
+        //      .verify(&pks[..], certificate_digest.as_ref())
+        //      .map_err(|_| DagError::InvalidSignature)?;
```

Comment:
For ref: here is where the aggregate sig would be useful.

Reply:
Howard said don't bother, just loop over them